src/atoms/TextInput/Text.web.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AtomText from '../Text';
+import Space from '../Space';
+import View from '../View';
+import Flex from '../Flex';
+import isEmpty from '../../_helpers/isEmpty';
+
+const styles = {
+  color({ disabled, errorText }) {
+    if (disabled) {
+      return 'shade.930';
+    } else if (!isEmpty(errorText)) {
+      return 'negative.900';
+    } else {
+      return 'shade.950';
+    }
+  },
+};
+
+const Text = ({ helpText, errorText, disabled }) => {
+  const displayText = isEmpty(errorText) ? helpText : errorText;
```

Comment:
This check is slightly confusing to read. Possible to simplify it to `errorText || helpText`? Check if it's the intended behaviour and covering edge cases.